src/plugin-styles.scss

```diff
@@ -0,0 +1,30 @@
+$plugin-prefix: "create-block-theme";
+$modal-footer-height: 70px;
+
+.#{$plugin-prefix} {
+	&__metadata-editor-modal {
+		width: 65vw !important;
```

Comment:
If you define the width with CSS, It would be better to add a media query (`@media (min-width: 782px)`). See: https://github.com/WordPress/gutenberg/pull/54518

Reply:
`size="large"` seems to be fitting well.